toqito/channel_metrics/fidelity_of_separability.py

```diff
@@ -0,0 +1,198 @@
+                \Gamma^{\mathcal{E}^{k}}_{RA^{\prime k}}=\mathcal{P}_{A^{\prime k}}(\Gamma^{\mathcal{E}^{k}}_{RA^{\prime k}}),\\ 
+                T_{A^{\prime}_{1\cdots j}}(\Gamma^{\mathcal{E}^{k}_{RA^{\prime k}}}) \geq 0 \quad \forall j\leq k
+        \end{array}\right\}
+        \end{multline}
+
+    :math:`\frac{1}{2}(1+\widetilde{F}_s^2(\rho_{AB}))` is the quantity to be
+    approximated but this function returns
+    :math:`\widetilde{F}_s^2(\rho_{AB})`.
+
+    :math:`\operatorname{Tr}[\Pi_{A^{\prime}A}^{\operatorname{sym}} \operatorname{Tr}_{R}[T_R(\psi_{RAB})\Gamma^{\mathcal{E}^{k}}_{RA^{\prime}_1}]]`
+    is the maximization problem subject to PPT & k-extendibile channel
+    constraints.
+
+    :math:`\Gamma^{\mathcal{E}^{k}}_{RA^{\prime}}` is Choi operator of
+    entanglement breaking channel :math:`\mathcal{E}^{k}`.
+
+    :math:`\mathcal{P}_{A^{\prime k}}` is the permutation operator over
+    k-extensions :math:`A^{\prime k}`.
+
+    The other constraints are due to the PPT condition [Per96]_.
+
+    Examples
+    ==========
+    Let's consider a density matrix of a state that we know is pure &
+    separable. :math:`|000 \rangle = |0 \rangle \otimes |0 \rangle \otimes |0 \rangle`.
+    
+    The expected approximation of fidelity of separability is the maximum
+    value possible i.e. very close to 1.
+
+    .. math::
+        \rho_{AB} = |000 \rangle \langle 000|
+
+    .. code-block:: python
+
+        from toqito.channel_metrics import fidelity_of_separability
+        from toqito.matrix_ops import tensor
+        from toqito.states import basis
+
+        state = tensor(basis(2, 0), basis(2, 0), basis(2, 0))
+        rho = state*state.conj().T
+        expected_value = fidelity_of_separability(rho, [2,2,2])
+
+    >>> expected_value = 0.9999999979949119
+
+
+    References
+    ==========
+    .. [Hay12] Hayden, Patrick et.al.
+        "Two-message quantum interactive proofs and the quantum separability problem."
+        Proceedings of the 28th IEEE Conference on Computational Complexity, pages 156-167.
+        https://arxiv.org/abs/1211.6120
+
+    .. [Per96] Peres, Asher.
+        "Separability Criterion for Density Matrices"
+        https://arxiv.org/abs/quant-ph/9604005
+
+    .. [Phil23] Philip, Aby et.al.
+        "Quantum Steering Algorithm for Estimating Fidelity of Separability"
+        https://arxiv.org/abs/2303.07911
+
+    .. [TBWat18] Watrous, John.
+        “The Theory of Quantum Information”
+        Cambridge University Press, 2018
+
+    Args:
+        psi: the density matrix for the tripartite state of interest psi_{BAR}
+        psi_dims: the dimensions of System A, B & R in
+            the input state density matrix. It is assumed that the first
+            quantity in this list is the dimension of System B.
+        k: value for k-extendibility
+    Raises:
+        AssertionError:
+            * If the provided dimensions are not for a tripartite density \n
```

Comment:
Hmm, is this `\n` needed?

Reply:
Yes, if I enforce the lentgh of a line has only 79 characters. But we could disregard these in the docstrings, maybe?

Reply:
Oh, I mean that I don't think you need the explicit `\n` character. Yes, you'll need a line break, but (I don't believe) including `\n` is necessary, right?